src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1279,57 +1296,69 @@ To make the model even more efficient, a transformed data variable
 defined to be `sum(y)` could be used in the place of `sum(y)`.
 
 
-## Standardizing predictors and outputs
+## Standardizing predictors
 
-Stan programs will run faster if the input is standardized to have a
+Stan programs will run faster if the data is standardized to have a
 zero sample mean and unit sample variance.  This section illustrates
 the principle with a simple linear regression.
 
 Suppose that $y = (y_1,\dotsc,y_N)$ is a sequence of $N$ outcomes and
-$x = (x_1,\dotsc,x_N)$ a parallel sequence of $N$ predictors.  A
-simple linear regression involving an intercept coefficient $\alpha$
-and slope coefficient $\beta$ can be expressed as
+$X$ is a matrix of size $N \times J$ where $J$ is the number of predictors.
+Each column $j$ of $X$ represents a feature vector $\textbf{x}_j$.
+
+A linear regression involving an intercept coefficient $\alpha$
+and an $J$-length vector of slope coefficients $\mathcal{B}$ can be expressed as
 $$
-y_n = \alpha + \beta x_n + \epsilon_n,
+y_n = \alpha + \mathcal{B} X[n, ] + \epsilon_n,
```

Comment:
For consistent mathematical notation,

* \mathcal{B} -> \beta
* X[n, ] -> X_n -OR- X_{n, \cdot}

Or if you want to do it in Stan code,

```stan
y[n] = alpha + beta * X[n] + epsilon[n];
```